### ISSUE TITLE
kitinerary not able to extract information from PDF files

### ISSUE THREAD

**Issue by TraceyC77** [MCMic/workflow_kitinerary#40]
I set up a new kitinerary flowusing a tag "kitinerary", set to act when files are saved, updated, accessed or the kitinerary tag is added

The flow is not working. I have saved a couple of travel related emails to PDF files. One is a parking reservation, the other is flight information. They are saved in a folder with the "kitinerary" tag. I also added the "kitinerary" tag through the Nextcloud files page. After a couple of minutes, I refreshed my calendar page, but neither thing showed up.

I see errors related to kitinerary in the logs, attached. It starts with
```
[webdav] Error: Error: Call to a member function getIterator() on null at <<closure>>
```

[kitinerary_error_log.txt](https://github.com/MCMic/workflow_kitinerary/files/10473281/kitinerary_error_log.txt)

_Originally posted by @TraceyC77 in https://github.com/MCMic/workflow_kitinerary/issues/32#issuecomment-1399350041_

**Comment by MCMic** [comment]
What happens if you pass those files to kitinerary-extractor in cli? Does it find anything?

**Comment by MCMic** [comment]
The error itself will be fixed by https://github.com/MCMic/workflow_kitinerary/pull/41 but still no event will be added to the calendar if kitinerary does not extract anything, which is what I suspect is happening.

**Comment by TraceyC77** [comment]
Thanks for your patience. I've tested kitinerary-extractor on command line, and your suspicion is correct. The extractor is not extracting anything from the PDFs I saved from united.com with my itinerary. It was able to extract data from the testing sample from the KItinerary project
https://github.com/KDE/kitinerary/raw/master/autotests/extractordata/synthetic/iata-bcbp-demo.pdf

That event does appear on Sept 6, 2019 on my Nextcloud calendar, so the flow is working properly.
